monitor/redis/watcher.go

```diff
@@ -71,11 +71,16 @@ func (w *Watcher) monitor(ctx context.Context, ch chan<- []*change.Change) {
 }
 
 func (w *Watcher) getValues(ctx context.Context, ch chan<- []*change.Change) {
-	values, err := w.client.MGet(ctx, w.keys...).Result()
-	if err != nil {
-		log.Errorf("failed to MGET keys %v: %v", w.keys, err)
-		return
+	values := make([]interface{}, len(w.keys))
```

Comment:
We should be safe to use `string` here because `Get(...).Result()` returns a string. It would reduce the complexity and prevent from casting to `interface{}` and then again to `string`